project/src/utils/common.ts

```diff
@@ -20,11 +24,64 @@ function adaptToClient(film: FilmFromServerType): Film {
       'description': film.description,
       'director': film.director,
       'starring': film.starring,
+      'scoresCount': film.scores_count,
     };
   return adaptedFilm;
 }
 
 const isCheckedAuth = (authorizationStatus: AuthorizationStatus): boolean =>
   authorizationStatus === AuthorizationStatus.Unknown;
 
-export {adaptToClient, isCheckedAuth};
+const getCurrentFilm = (filmsArray: Film[], id: string): any =>
+  filmsArray.find((filmItem) => filmItem.id === Number(id)) || ({} as Film);
```

Comment:
```suggestion
  filmsArray.find((filmItem) => filmItem.id === Number(id)) ?? null;
```
лучше никогда так не делай. это дальше будет максимально путать. возвращай null, если ничего не нашло, и обрабатывай там где нужно